Reconstruction/reconstruction.tex

```diff
@@ -343,7 +344,7 @@ \subsection{Hadrons}
 Afterwards, the only remaining PF elements are HCAL clusters linked to one or more tracks and ECAL clusters linked to one of these tracks.
 A single charged hadron is constructed for each remaining HCAL cluster, with energy equal to the sum of the ECAL and HCAL clusters and momentum equal to the sum of the individual track momenta.
 
-If the energy of the charged hadron exceeds its momentum by an amount larger than the calometric energy resolution, neutral hadrons and photons are added.
+If the energy of the charged hadron exceeds its momentum by an amount larger than the calometric energy resolution, neutral hadrons and photons are added. \PH{This paragraph starts out of nowhere, was hard to follow reading it, perhaps merge with previous paragraph and rephrase a little}
```

Comment:
added reasons why energy != momentum in the previous paragraph and then started with the equals case. idk why PF paper did it ini this order